.github/workflows/main.yml

```diff
@@ -32,15 +40,13 @@ jobs:
         if: ${{ failure() }}
         run: yarn diff:mdx && echo "::error::Prettier static analysis failed"
 
-      - name: Copy Files From Stellar CLI
-        run: yarn stellar-cli:build
-
-      - name: Build app
-        run: yarn build
-
       - name: Check if routes.txt file needs updating
         run: |
           if ! git diff --exit-code -- routes.txt; then
             echo 'The `routes.txt` file has changed. Please update the routes.txt file locally by running `yarn build` and push that change.'
             exit 1
           fi
```

Comment:
There's no point checking the routes.txt here because it's not updated when not running build.